### DISCUSSION TITLE
Please tell me your favorite game players!

### DISCUSSION THREAD

**Discussion by WeiqiZhang1999** [NAIST-SE/AttractingContributors#409] [General]
If you play games, I think you maybe have watched Professional Matches.
So please tell me your favorite Pro players!
Here is reference for League of Legends: https://www.trackingthepros.com/players/
But any game is OK!

My examples:
1. Xiaohu(League of Legends)
2. Gala(League of Legends)
3. ZywOo(Counter Strike Global Offensive)

My fork: https://github.com/WeiqiZhang1999/AttractingContributors_Games

**Comment by aru1702** [comment]
I don't know so much about pro player gamers.
I will add with old player that I know.
See my repo >> https://github.com/aru1702/AttractingContributors-TouhouFumo <<

**Reply by WeiqiZhang1999** [reply]
Don't mind, I will pull request later.

**Comment by Kotjik** [comment]
Hey, I contributed to your repo - would you mind adding something to mine as well? 🙂
Here's my repo: https://github.com/Kotjik/AttractingContributors-FavoriteCharacter

**Reply by WeiqiZhang1999** [reply]
Sure! Here I go.

**Comment by vincentmichael089** [comment]
Hello! I contributed to your repository; is it okay for you to also contribute to my repository? Thanks in advance!
https://github.com/vincentmichael089/AttractingContributors-Kanji

**Comment by suika10000** [comment]
Update with pro players.
This play was very impressive.
https://twitter.com/noppo_cs/status/601038596854796288

If you have time, please add a note to my repository.
Here's my repo: https://github.com/suika10000/AttractingContributors-GPU
